nodebuilder/fraud/module.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/celestiaorg/celestia-node/nodebuilder/node"
 )
 
-var log = logging.Logger("fraud-module")
+var log = logging.Logger("constructor/fraud")
```

Comment:
The constructor is confusing. 
1. Constructor of the fraud references the fraud pkg more
2. This pkg could log out things beyond the module construction.

Let's do `modfraud` or `fraudmod` pls

Reply:
So, the reasoning behind this is that the user won't know what a "nodebuilder" is nor will understand what the difference between "fraud" and "modfraud" packages would be. I like the constructor pattern but am open for more ideas. Not `modX` pilled yet, do you have any other ideas?

Reply:
The point is that the logger name should reference the fact that it is a module. The `constructor` does not do that.

Reply:
Why not just `modfraud` or `mod/fraud`?